app/views/test/components/common/markdown_editor/show.html.haml

```diff
@@ -0,0 +1 @@
+= ReactComponents::Common::MarkdownEditor.new(SecureRandom.uuid)
```

Comment:
What's the purpose of the `uuid` here? If it's just random each time, then this should be moved into the `Component` class. 

```suggestion
= ReactComponents::Common::MarkdownEditor.new(SecureRandom.uuid)
```

Reply:
I think we should configure the id per context, so it reads better when viewing the keys on localstorage. I just generated a random id for here as this is just a test page.